packages/schematics/src/ng-add/index.ts

```diff
@@ -29,6 +29,7 @@ function addAngularESLintPackages() {
     const json = JSON.parse(projectPackageJSON);
     json.devDependencies = json.devDependencies || {};
     json.devDependencies['eslint'] = packageJSON.devDependencies['eslint'];
+    json.scripts['lint'] = json.scripts['lint'] || 'ng lint';
```

Comment:
I think it's worth being defensive and ensuring scripts exists before attempting to add to it, like we do with the deps

Reply:
Good point! Commit squashed and fixed in f348f05e14f0eee1652085d74e47830ab4ab9f60.